src/main/java/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import java.util.List;
+
+public interface FileService {
+    List<String> read(String filePath);
+
+    void write(String reportPath, String report);
```

Comment:
I believe this service can write to file Happy New Year greetings as well. Remember about **abstraction**
```suggestion
    void write(String filePath, String text);
```

Reply:
I checked service could wrote wishes ) 
What wrong whith this, i can`t understand... 
string number 8 was coreccted.

Reply:
Смысл в том, что имена переменных со словом report будут смотреться странно, если мы начнем использовать этот интерфейс в другом месте для записи в файл какого-то другого текста. Мы всегда должны помнить про абстракцию в нашем коде. Метод умеет писать текст (любой) в файл (тоже любой). Методу не важно, что там за текст. Так и нужно назвать переменные, не привязываясь к одной конкретной имплементации или одному конкретному месту использования.

Reply:
поняла, спасибо!